tmt/steps/execute/internal.py

```diff
@@ -111,7 +116,6 @@ def wake(self, keys=None):
     def _show_progress(self, progress, test_name, finish=False):
         """
         Show an interactive progress bar in non-verbose mode.
-
```

Comment:
Probably accidentaly removed blank line from the docstring

Reply:
I've fixed this one too.